app.py

```diff
@@ -108,9 +108,9 @@ def render_content(tab):
         layout tab
     """
     if tab == 'tab-1':
-        return parameters.tab_1_layout
-    elif tab == 'tab-2':
         return current.tab_2_layout
```

Comment:
is a bit confusing, maybe change the variable in the parameters and current file to just tab_layout instead of tab_1_layout and tab_2_layout?

Reply:
Done

Reply:
Great :)